src/resolvers/header.resolver.ts

```diff
@@ -31,7 +31,7 @@ export class HeaderResolver implements I18nResolver {
             'HeaderResolver does not support RFC4647 Accept-Language header. Please use AcceptLanguageResolver instead.',
           );
         }
-        if (req.headers[key] !== undefined) {
+        if (req.headers != undefined && req.headers[key] !== undefined) {
```

Comment:
Small change needed, `req.headers !== undefined` instead of `!=`

Reply:
Thank you for the review. This is what I intended to write but it seems it slipped my attention.